media: i2c: Modify the datatype of PDAF data in the arducam_64mp driver

ArduCAM opened the pull request:
Modify the PDAF Datatype of the Arducam 64MP camera from 0x30 to 0x12 so that the Raspberry Pi 5 cfe driver can receive PDAF data.

6by9 commented:
Matches the information I have. LGTM once CI has run.
I'm not fussed over the checkpatch warning for author and SoB not matching as I'm certainly not intending to upstream this driver.

Commit:
media: i2c: Modify the datatype of PDAF data in the arducam_64mp driver

Modify the PDAF Datatype of the Arducam 64MP camera from 0x30 to 0x12
so that the Raspberry Pi 5 cfe driver can receive PDAF data.

Signed-off-by: Lee Jackson <info@arducam.com>
